### PR TITLE
update dashboard

### PR DESCRIPTION
**Pull request by tvqphong001**
test pull request

### CODE REVIEW

**Comment by PhongTranWee** [comment]
it looking good

**Comment by phucho2306** [comment]
hh

**Review by phucho2306 — COMMENTED** [review]
ok oke

**Review by PhongTranWee — APPROVED** [review]
ok

**Review by phucho2306 — APPROVED** [review]

**Review by PhongTranWee — APPROVED** [review]

**Review by phucho2306 — APPROVED** [review]

**Review by PhongTranWee — APPROVED** [review]
aa

**Review by PhongTranWee — APPROVED** [review]

File: lib/modules/screen/dashboard_screen.dart [hunk 1]
```diff
@@ -0,0 +1 @@
+//update dashboard
```
Comment (PhongTranWee) [inline lib/modules/screen/dashboard_screen.dart:1]: hi hi

**Review by PhongTranWee — APPROVED** [review]

File: lib/modules/screen/dashboard_screen.dart (hunk 1 above)
Comment (PhongTran004) [inline lib/modules/screen/dashboard_screen.dart:1]: ok it

**Review by PhongTran004 — APPROVED** [review]

**Review by PhongTran004 — APPROVED** [review]

**Comment by tvqphong001** [comment]
ok

**Review by PhongTranWee — APPROVED** [review]

**Pull request merged**